ArcGISRuntimeSDKQt_QMLSamples/DisplayInformation/Simple_Renderer/README.metadata.json

```diff
@@ -0,0 +1,37 @@
+{
+    "category": "Display Information",
```

Comment:
```suggestion
    "category": "Display information",
```

The punctuation of category names matters for the nav bar: 

![image](https://user-images.githubusercontent.com/46973965/57724835-830a8500-7640-11e9-926e-8a8b6de45e2f.png)